src/app/page.module.css

```diff
@@ -1,9 +1,89 @@
-.main {
+.mainpageContainer {
+  background: #1b1228;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 6rem;
-  min-height: 100vh;
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding: 0;
+}
+
+.h1 {
+  width: 353px;
+  height: 92px;
+  top: 233px;
+  left: 19px;
+  gap: 0px;
+  opacity: 0px;
+  font-family: Cascadia Mono;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 46.48px;
+  text-align: center;
+  color: #ffffff;
+  margin-bottom: 100px;
+}
+.headerconatiner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 250px;
+}
+.lgnbttncontainer {
+  display: flex; /* Flexbox for centering */
+  justify-content: center;
+  align-items: center;
+  margin-top: 100px; /* Adjust as needed */
+  width: 324px; /* Fixed width */
+  height: 48px; /* Fixed height */
+  padding: 1px;
+  gap: 10px;
+  opacity: 1; /* Make the container visible */
+  border: none;
+}
+
+.loginbtton {
+  width: 324px;
+  height: 48px;
+  background: #b077ff;
+  font-family: Cascadia Mono;
+  color: #ffffff;
+  font-size: 28px;
+  font-weight: 400;
+  line-height: 27.89px;
+  text-align: center;
+  border-radius: 5px;
+  border: none;
+  cursor: pointer;
+}
+
+.signbttncontainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 50px;
+  width: 324px;
+  height: 48px;
+  padding: 1px;
+  gap: 10px;
+  opacity: 1;
+  border: none;
+}
+
+.signupbtton {
+  width: 324px;
+  height: 48px;
+  background: #0c0811;
+  font-family: Cascadia Mono;
+  color: #ffffff;
+  font-size: 28px;
+  font-weight: 400;
+  line-height: 27.89px;
```

Comment:
no need to specifiy font since these are defined in globals.css. defining it here will override the styling from globals.css and we want to avoid that

Reply:
the font turned way to small when i removed these

Reply:
thats okay, on globals.css your gonna have to set a fixed font size for the button element through CSS and it will fix it across all buttons. i would avoid setting font-weight and line-height for now since those should rarely ever be changed